reconcile/deadmanssnitch.py

```diff
@@ -32,6 +34,17 @@
 SECRET_NOT_FOUND = "SECRET_NOT_FOUND"
 
 
+class ClusterFields(BaseModel):
+    """Class to hold values from cluster file and settings"""
+
+    prometheus_url: str
```

Comment:
Could use `SnitchSpec` to indicate it's desired, and the value of `prometheus_url` is no longer a url but a name already, right? Can use `name`.